globals/card_definitions.gd

```diff
@@ -923,6 +964,8 @@ func get_effect_type_text(effect, card_name_source : String = "", char_effect_pa
 				effect_str += "+" + str(effect['amount']) + " Speed per EVERY boost in play."
 			else:
 				effect_str += "+" + str(effect['amount']) + " Speed per boost in play."
+		"spend_all_gauge_and_save_amount":
+			effect_str += "Discard all (X) cards in gauge"
```

Comment:
why X?
Isn't it just all?

Reply:
kind of an artifact from an older implementation whoops